lib/MemoryAnalysisModules/LoopVariantAllocation.cpp

```diff
@@ -60,7 +60,7 @@ static bool isNoaliasWithinLoop(const Value *src, const Loop *L,
                 0, Attribute::NoAlias))
           return true;
 
-      if (isNoAliasCall(src))
+      if (isNoAliasCall(src) || isAllocationFn(src, &tli))
```

Comment:
Put associate commit here (not sure if this is right)

Reply:
Same change as in FindSource.cpp, this is a change I made while upgrading. I can add a FIXME and/or comment.